src/darwin/Framework/CHIP/MTROTAProviderDelegateBridge.mm

```diff
@@ -574,11 +597,18 @@ bool GetPeerNodeInfo(CommandHandler * commandHandler, const ConcreteCommandPath
                           CHIP_ERROR err = gOtaSender.PrepareForTransfer(fabricIndex, nodeId);
                           if (CHIP_NO_ERROR != err) {
                               LogErrorOnFailure(err);
-                              handler->AddStatus(cachedCommandPath, Protocols::InteractionModel::Status::Failure);
-                              handle.Release();
-                              return;
+                              if (err == CHIP_ERROR_BUSY) {
+                                  response.status = static_cast<OTAQueryStatus>(MTROTASoftwareUpdateProviderOTAQueryStatusBusy);
+                                  handler->AddResponse(cachedCommandPath, response);
```

Comment:
This is not right.  This will take the response we got from the delegate, which for example has an ImageURI, and will send it out with just the status changed.  But those fields should not appear here if the status is not UpdateAvailable.

What we should be doing here is creating a new `Commands::QueryImageResponse::Type`, setting its status, setting its `delayedActionTime` from `response`, then if that ended up null setting some time we pick, because for a BUSY `delayedActionTime` is _required_.  Then not set any other fields, probably, and send it.

Reply:
Hmm. That sounds good. will fix.

Reply:
done